dallinger/experiment.py

```diff
@@ -309,6 +309,20 @@ def add_node_to_network(self, node, network):
         """
         network.add_node(node)
 
+    def check_entry_information(self, entry_information):
```

Comment:
I'm not sure from looking at the code and comments what I'm supposed to do when the information is invalid, so we should document this. Raise a `ValueError`? Custom exception type?

(See also later comment about the name.)

Reply:
Returning a dict with an unset `assignment_id` is probably best, since the routes already handle that case.